function-app-container/README.md

```diff
@@ -0,0 +1,37 @@
+# GitHub Action for deploying containers on Azure Function App
+
+
+## Usage
+
+```
+
+action "Deploy containers on Azure Function App" {
+  uses = "actions/azure/functionscontainer@master"
```

Comment:
will this work? are the actions published in github actions repo ? same pattern has been used at other places.

Reply:
I think the action folder is named as function-app-container. @vincentdass

Reply:
resolving this in favor of above comment